docker-compose.yml

```diff
@@ -10,6 +11,8 @@ services:
       - ./traefik.yaml:/etc/traefik/traefik.yaml:ro
 
   frontend:
+    image: yeagerai/simulator-frontend:${STUDIO_DOCKER_IMAGE_VERSION:-latest}
+    pull_policy: build
     build:
```

Comment:
I think we should remove the `build` and `volumes` sections, since they are not needed anymore